core/core.js

```diff
@@ -233,7 +231,13 @@ SharedQueue Binary Layout
     if ("ok" in res) {
       return res.ok;
     } else {
-      throw new (getErrorClass(res.err.className))(res.err.message);
+      const ErrorClass = getErrorClass(res.err.className);
+      if (!ErrorClass) {
+        throw new Error(
+          `Unregistered Deno core error class: ${res.err.className}\n  ${res.err.message}`,
```

Comment:
`Unregister error class: "<error_class>"\n <message>\n Did you forget to register error using Deno.core.registerErrorClass?`

Reply:
Only thought about this is the is it really going to often not the end-user who did this, so stating "Did you forget..." is highly unlikely...  I will try some other words.